src/AltinnCore/Designer/Controllers/DeploymentsController.cs

```diff
@@ -0,0 +1,52 @@
+using System.Threading.Tasks;
+using AltinnCore.Designer.ModelBinding.Constants;
+using AltinnCore.Designer.Repository.Models;
+using AltinnCore.Designer.Services;
+using AltinnCore.Designer.ViewModels.Request;
+using AltinnCore.Designer.ViewModels.Response;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace AltinnCore.Designer.Controllers
+{
+    /// <summary>
+    /// Controller for creating, getting and updating deployments
```

Comment:
We can remove "updating" from the controller as there is no PUT-method

Reply:
And done :)